Design.md

```diff
@@ -162,6 +163,18 @@ Originally discussed in [#21](https://github.com/coreos/fedora-coreos-tracker/is
 We will identify a Fedora CoreOS server using the `ID=fedora` and `VARIANT_ID=coreos`
 fields in the `/etc/os-release` file.
 
+## Firewall management
+
+Originally discussed in [#26](https://github.com/coreos/fedora-coreos-tracker/issues/26).
+
+### Summary:
+
+ - FCOS will ship without any ad-hoc filtering rules. By default, nodes will boot without firewall.
+ - Kernel- and user-space components will be provided for both iptables and nft.
```

Comment:
should we explicitly state what software pieces we are delivering here, so there won't be any confusion?

Reply:
mainly the names of the rpms we will include.. would that be `iptables` and `nftables` ?

Reply:
I was wanting to make sure someone couldn't point at this statement and say "you said you would include `$RANDOM_FIREWALL_TOOL`".

Reply:
I see. Then yes, let me reword this entry in that direction. The set of packages should be `iptables`, `nftables`, `iptables-nft` (for the "restore" bits).

Reply:
did you update it yet?